qcodes/tests/__init__.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from qcodes.config import Config
+
+
+@pytest.fixture(scope="session", autouse=True)
+def disable_telemetry():
+    """
```

Comment:
Do we really need to write to home? Can we not just modify the `qcodes.config` instance in memory. Alternatively could we save to current dir or use the env variable to load a custom file

Reply:
That dreaded `qc.config`... Let me look at it. I agree that producing files on disk is not a nice side effect of running the tests. And of course, a file in the current dir could even break the effect of this fixture.